package.json

```diff
@@ -3,12 +3,18 @@
   "version": "0.1.0",
```

Comment:
- [ ] Kindly change the name in your `package.json` file from `math-magicians` to `React todo`  because you are currently working on the todo app and not the math magicians app, it is best practice to always name your project accordingly.